### PR TITLE
GT: Add PLDM OEM event log for VR alert

### PR DESCRIPTION
**Pull request by Quanta-YangChen**
Summary:
- Add event class for sending PLDM OEM event.
- Add VR alert OEM event with one-byte event sensor ID and two-byte status word.

Test Plan:
- Build code: Pass

### CODE REVIEW

**Review by amithash — CHANGES_REQUESTED** [review]
OEM commands are frowned upon. Can you provide more information on why this cannot be accomplished using standard commands and what other standard options were evaluated.

File: common/service/pldm/pldm_monitor.c [hunk 1]
```diff
@@ -431,6 +431,10 @@ uint8_t pldm_send_platform_event(uint8_t event_class, uint16_t id, uint8_t ext_c
 		return pldm_send_effecter_event_message(event_receiver_info_p->mctp_inst_p,
 							event_receiver_info_p->ext_params, id,
 							ext_class, event_data, event_data_length);
+	case PLDM_OEM_EVENT_START ... PLDM_OEM_EVENT_END:
```
Comment (williamspatrick) [inline common/service/pldm/pldm_monitor.c:434]: This is a GCC extension that we should probably refrain from using.

**Comment by Quanta-YangChen** [comment]
> OEM commands are frowned upon. Can you provide more information on why this cannot be accomplished using standard commands and what other standard options were evaluated.

File: common/service/pldm/pldm_monitor.c (hunk 1 above)
Comment (Quanta-YangChen) [reply common/service/pldm/pldm_monitor.c:434]: Ok, thanks for the suggestion, will avoid using the extension like this in the future.

**Pull request closed without merging**